1.8/administration/installing/cloud/aws/upgrading.md

```diff
@@ -0,0 +1,42 @@
+---
+post_title: Upgrading
+menu_order: 11
+---
+
+## Summary
+
+This document provides instructions for upgrading a DC/OS cluster from version 1.7 to 1.8 using AWS cloudformation templates. It is recommended that you familiarize yourself with the [Advanced DC/OS Installation on AWS] before proceeding.
+
+If this upgrade is performed on a supported OS with all prerequisites fulfilled, this upgrade _should_ preserve the state of running tasks on the cluster.
+
+**Important:**
+
+- The Advanced Installation method is the _only_ recommended upgrade path for DC/OS. It is recommended that you familiarize yourself with the [Advanced DC/OS Installation Guide][advanced-install] before proceeding.
+- The [VIP features](/docs/1.8/usage/service-discovery/load-balancing-vips/virtual-ip-addresses/), added in DC/OS 1.8, require that ports 32768 - 65535 are open between all agent and master nodes for both TCP and UDP.
+- The DC/OS UI and APIs may be inconsistent or unavailable while masters are being upgraded. Avoid using them until all masters have been upgraded and have rejoined the cluster. You can monitor the health of a master during an upgrade by watching Exhibitor on port 8181.
+- Task history in the Mesos UI will not persist through the upgrade.
+
+## Instructions
+
+1. Login to the current leader master of the cluster.
+   1. Using dcos-cli: 
+      ./dcos node ssh --master-proxy --leader  
+   1. Once logged in , run the following command, this creates the new 1.8 directory /var/lib/dcos/exhibitor/zookeeper as a symlink to the old /var/lib/zookeeper: 
+      ```
+      for node in $(dig +short master.mesos); do ssh -o StrictHostKeyChecking=no $node "sudo mkdir -p /var/lib/dcos/exhibitor && sudo ln -s /var/lib/zookeeper /var/lib/dcos/exhibitor/zookeeper"; done
+      ```
+1. Login to the DC/OS dashboard
+   1. Go to dcos-dashboard-ip/exhibitor 
+   1. Go to config tab
+   1. Edit the config and change all three fields that contain /var/lib/zookeeper/ to /var/lib/dcos/exhibitor/zookeeper/
+   1. Commit and perform a rolling restart
+   1. This will take a couple of minutes and during that time the Exhibitor UI will flash, wait the commit to be performed fully
+   1. Make sure the cluster is healthy at this point
+   
+1. Updating cloudformation stacks
+   1. Generate the new templates following instructions at https://docs.mesosphere.com/1.8/administration/installing/cloud/aws/advanced/aws-custom/
```

Comment:
This should be a relative link. There are examples of relative links to other docs in the repo.